master/getting-started/kubernetes/tutorials/app-layer-policy/index.md

```diff
@@ -0,0 +1,298 @@
+---
+title: Application layer policy tutorial
+canonical_url: 'https://docs.projectcalico.org/master/getting-started/kubernetes/tutorials/app-layer-policy/'
+---
+
+The included demo sets up a microservices application, then demonstrates the use of application
+layer policy to mitigate some common threats.
+
+## Pre-requisites
+
+To create a Kubernetes cluster which supports application layer policy, follow
+one of our [getting started guides]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes),
+and [enable application layer policy]({{site.baseurl}}/{{page.version}}/getting-started/kubernetes/installation/app-layer-policy).
+
+You will also need the [calicoctl]({{site.baseurl}}/{{page.version}}/usage/calicoctl/install) command line tool.
+
+### Install the demo application
+
+We will use a simple microservice application to demonstrate Calico application layer policy.  The 
+[YAO Bank] application creates a customer-facing web application, a microservice that serves up
+account summaries, and an [etcd] database.
```

Comment:
s/database/datastore

Reply:
I'm not sure the distinction you're trying to draw with this one.